sample-apps/discounts-tutorial/web/frontend/App.jsx

```diff
@@ -21,12 +21,7 @@ export default function App() {
           <DiscountProvider>
             <QueryProvider>
               <NavigationMenu
-                navigationLinks={[
-                  {
-                    label: "New volume discount",
-                    destination: "/Volume/new",
-                  },
-                ]}
```

Comment:
Negative. Will get this cleaned up!

Reply:
Although just remembered that this might not be following our design principals for navigation and since these are `primary navigation` - these links might not make sense.  I'll double check with @andrewhassan though, IIRC that's why I originally removed.